### ISSUE TITLE
Custom Field Creation. 

### ISSUE THREAD

**Issue by jonathansimmons** [mattbeedle/capsule_crm#64]
I'm having trouble working out how to add a custom field on person and organisation. 

This only way I've found that works is to add them after the person is created: 

```
person = CapsuleCRM::Person.create(first_name: "Jonathan", last_name: "Simmons")
person.custom_fields << CapsuleCRM::CustomField.new(label: 'field_name', text: 'field_value')
person.save
```

The above method does not work when trying to add custom_fields to an organization though...

My preferred method would be to add the custom field inline with the create like so: 

<pre>
 person = CapsuleCRM::Person.create(
    first_name: "Jonathan", 
    last_name: "Simmons", 
    custom_fields: CapsuleCRM::CustomField.new(
         label: 'field_name', 
         text: 'field_value'
    )
  )
</pre>

This returns the following error: 

```
NoMethodError: undefined method `map' for #<CapsuleCRM::CustomField:0x007fdea4188820>
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/capsule_crm-1.5.2/lib/capsule_crm/serializer.rb:20:in `serialize_collection'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/capsule_crm-1.5.2/lib/capsule_crm/associations/has_many_proxy.rb:44:in `to_capsule_json'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/capsule_crm-1.5.2/lib/capsule_crm/associations/has_many_proxy.rb:48:in `save'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/capsule_crm-1.5.2/lib/capsule_crm/associations/has_many.rb:47:in `block in has_many'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/activesupport-3.2.17/lib/active_support/callbacks.rb:405:in `_run__2604789604786209564__save__2761873270767723794__callbacks'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/activesupport-3.2.17/lib/active_support/callbacks.rb:405:in `__run_callback'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/activesupport-3.2.17/lib/active_support/callbacks.rb:385:in `_run_save_callbacks'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/activesupport-3.2.17/lib/active_support/callbacks.rb:81:in `run_callbacks'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/capsule_crm-1.5.2/lib/capsule_crm/persistence/persistable.rb:25:in `save'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/capsule_crm-1.5.2/lib/capsule_crm/persistence/persistable.rb:15:in `tap'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/capsule_crm-1.5.2/lib/capsule_crm/persistence/persistable.rb:15:in `create'
from (irb):12
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/railties-3.2.17/lib/rails/commands/console.rb:47:in `start'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/railties-3.2.17/lib/rails/commands/console.rb:8:in `start'
from /Users/jonathan/.rvm/gems/ruby-1.9.3-p392/gems/railties-3.2.17/lib/rails/commands.rb:41:in `<top (required)>'
from script/rails:6:in `require'
from script/rails:6:in `<main>'1.9.3p392 :013 > 
```

After reviewing #40 I tried the build method shown:

```
person.custom_fields.build(label: 'field_name', text: 'field_value')
person.save!
```

But that does not work on v 1.5.2 either.

If I can get some clarity on this I can make a pull request to update the readme with more examples.

To recap: What is the suggested method for adding a custom field? more than one at a time? And are they meant to function with organizations as they don't appear to now?
